astro/src/content/docs/release-notes/index.mdx

```diff
@@ -16,6 +16,25 @@ import ReleaseNoteHeading from 'src/components/docs/release-notes/ReleaseNoteHea
 
 Looking for release notes older than 1.23.0? Look in the [release notes archive](/docs/release-notes/archive). Looking to be [notified of new releases?](/docs/operate/roadmap/releases#release-notifications)
 
+<ReleaseNoteHeading version="1.48.2" releaseDate="November 21st, 2023" />
+
+### Fixed
```

Comment:
If you review existing release notes, you'll notice that when we mention the version we use backticks. 

I would suggest we keep doing this for consistency. Example: `1.48.0` - you have a few of these in the release notes.